api/pom.xml

```diff
@@ -19,13 +19,17 @@
     <parent>
         <groupId>org.eclipse.microprofile.metrics</groupId>
         <artifactId>microprofile-metrics-parent</artifactId>
-        <version>5.1.0-RC1</version>
+        <version>5.1.0-RC1.payara-p1</version>
     </parent>
 
     <artifactId>microprofile-metrics-api</artifactId>
     <name>MicroProfile Metrics API</name>
     <description>MicroProfile Metrics :: API</description>
 
+    <properties>
+        <bnd.baseline.continue.on.error>true</bnd.baseline.continue.on.error>
```

Comment:
Was this used only for development or is it still necessary?

Reply:
It's to bypass the BND tool complaining about the version number "not changing" despite there being changes - it doesn't understand our patch qualifier.